Layout w HTML i CSS/warmup/flexbox/style.css

```diff
@@ -0,0 +1,22 @@
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 200px;
```

Comment:
To, co wspomnieliśmy sobie live - 200px na Twoim ekranie nie da tego samego efektu, co wyświetlenie Twojej pracy na większym ekranie :)

Możesz pozbyć się tego wrappera i nadać

```
body {
  display: flex;
  justify-content: center;
  align-items: center;
}
```

Reply:
Tak zrobiłem ale musiałem dodać jeszcze wysokość body tak żeby centrowanie w pionie zadziałało.

body {
  display: flex;
  justify-content: center;
  align-items: center;
}